packages/sol/src/verification.ts

```diff
@@ -0,0 +1,372 @@
+import { FieldError } from "@litespace/types";
+import { sanitizeMessage } from "./chat";
+
+export function isValidEmail(email: string): FieldError.InvalidEmail | true {
+  const emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
```

Comment:
Please move any constant to `sol/src/constants.ts`. Constants should be all uppercase with `_` between words. Example: `USER_EMIAIL_REGEX`.